igor-web/src/main/groovy/com/netflix/spinnaker/igor/pubsub/PubsubSubscribers.groovy

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.pubsub
+
+import com.netflix.spinnaker.igor.model.PubsubType
+
+class PubsubSubscribers {
+  Map<String, PubsubSubscriber> map = new HashMap<String, PubsubSubscriber>()
```

Comment:
The other side of that internal state leakage commented on above.

Feels like this Bean could be created and provided a `List<PubsubSubscriber>` mebbe.